internal/util/util.go

```diff
@@ -115,15 +117,32 @@ func CheckDesiredBundleTemplate(existingBundle *rukpakv1alpha1.Bundle, desiredBu
 	bundle := existingBundle.DeepCopy()
 	delete(bundle.Labels, CoreOwnerKindKey)
 	delete(bundle.Labels, CoreOwnerNameKey)
-	return equality.Semantic.DeepEqual(bundle.Spec, desiredBundle.Spec) &&
-		equality.Semantic.DeepEqual(existingBundle.Annotations, desiredBundle.Annotations) &&
-		equality.Semantic.DeepEqual(bundle.Labels, desiredBundle.Labels)
+
+	if !equality.Semantic.DeepEqual(existingBundle.Annotations, desiredBundle.Annotations) ||
+		!equality.Semantic.DeepEqual(bundle.Labels, desiredBundle.Labels) {
+		return false
+	}
```

Comment:
Do we need any of these comparisons anymore? I would assume that we could include `template.Metadata` and `template.Spec` in our hash computation. And then all we need to do is compute the hash and see if it matches the pod's suffix.

Reply:
That sounds more reasonable -- thanks.